src/test/unit/ilp.test.js

```diff
@@ -51,19 +56,19 @@ describe('ILP', () => {
 
 
     test('ILP fulfilment should match condition', () => {
-        const {fulfilment, ilpPacket, condition} = ilp.getQuoteResponseIlp(quoteRequest, partialResponse);
+        const {fulfilment, condition} = ilp.getQuoteResponseIlp(quoteRequest, partialResponse);
 
-        const binaryPacket = Buffer.from(ilpPacket, 'base64');
-        const jsonPacket = IlpPacket.deserializeIlpPacket(binaryPacket);
-        console.log(`Decoded ILP packet: ${util.inspect(jsonPacket)}`);
+        // const binaryPacket = Buffer.from(ilpPacket, 'base64');
+        // const jsonPacket = IlpPacket.deserializeIlpPacket(binaryPacket);
+        // console.log(`Decoded ILP packet: ${util.inspect(jsonPacket)}`);
 
-        const dataElement = JSON.parse(Buffer.from(jsonPacket.data.data.toString('utf8'), 'base64').toString('utf8'));
+        // const dataElement = JSON.parse(Buffer.from(jsonPacket.data.data.toString('utf8'), 'base64').toString('utf8'));
 
-        console.log(`Decoded ILP packet data element: ${util.inspect(dataElement)}`);
+        // console.log(`Decoded ILP packet data element: ${util.inspect(dataElement)}`);
 
         const valid = ilp.validateFulfil(fulfilment, condition);
 
-        console.log(`Valudate fulfilment returned ${valid}`);
+        // console.log(`Valudate fulfilment returned ${valid}`);
```

Comment:
nit: spelling

Reply:
let don't keep commented code

Reply:
we shouldn't comment out tests without explanation in comments